README.md

```diff
@@ -1,26 +1,163 @@
+* <img src="https://github.com/BlockchainCommons/crypto-commons/blob/master/images/logos/crypto-commons-super-simple.png" width=16 valign="bottom">&nbsp;&nbsp; ***part of the [crypto commons](https://github.com/BlockchainCommons/crypto-commons/blob/master/README.md) technology family***
 
-## Displaying a Specific LifeHash
+### _by [Wolf McNally](https://www.github.com/wolfmcnally) and [Christopher Allen](https://www.github.com/ChristopherA)_
 
-You can access this page's URL with a query parameter to populate the string input field with a specific value. Example:
+<a href="https://lifehash.info"><img src="Nexus-Images/lifehash-info.png" align="right" width=400></a>
+This is a central repo for LifeHash, a method of visual hash visualization that creates beautiful, deterministic icons.
+<br><br>
 
-[http://lifehash.info?s=RBH-AOZ](http://lifehash.info?s=RBH-AOZ)
+It also contains the data for our [lifehash.info](https://lifehash.info) demonstration site for LifeHash, which uses our [reference C++ implementation of LifeHash](https://github.com/BlockchainCommons/bc-lifehash) compiled to [WebAssembly](https://webassembly.org/). The page itself can be accessed [here](http://lifehash.info).
 
-If you want to generate a LifeHash directly from a SHA-256 digest, you can use this form:
+## Overview
 
-[http://lifehash.info?f=70b0db749f1cb026282693b236048cefa3395fc580e91b7c62666b0bdc97c0e3](http://lifehash.info?f=70b0db749f1cb026282693b236048cefa3395fc580e91b7c62666b0bdc97c0e3)
+LifeHash is a method of hash visualization based on Conway's Game of Life that creates beautiful icons that are deterministic, yet distinct and unique given the input data.
+
+The basic concept is to take a SHA-256 hash of the input data (which can be any data including another hash) and then use the 256-bit digest as a 16x16 pixel "seed" for running the cellular automata known as [Conway’s Game of Life](https://en.wikipedia.org/wiki/Conway's_Game_of_Life).
+
+After the pattern becomes stable (or begins repeating) the resulting history is used to compile a grayscale image of all the states from the first to last generation. Using Game of Life provides visual structure to the resulting image, even though it was seeded with entropy.
+
+Some bits of the initial hash are then used to deterministically apply symmetry and color to the icon to add beauty and quick recognizability.
+
+View the LifeHash explainer video for more details:
+
+[![LifeHash Explainer on YouTube](Nexus-Images/title-card.jpeg)](https://www.youtube.com/watch?v=cu0K__KLxKo)
+
+See a LifeHash demo at [LifeHash.info](https://lifehash.info/).
+
+## Additional Information
+
+The following files contain more information about LifeHash:
+
+* [Use Cases](https://github.com/BlockchainCommons/lifehash.info/blob/shannona-lifehash-nexus/Nexus-Docs/Use-Cases.md) — Why use LifeHash?
```

Comment:
Looks like this URL is not in minimal form, like the others used in the list.

Reply:
Yep, changed.